plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/outputconsole/GitOutputConsolePresenter.java

```diff
@@ -35,6 +35,8 @@
 
     private final List<ConsoleOutputListener> outputListeners = new ArrayList<>();
 
+    private static final String RED_COLOR = "#F62217";
```

Comment:
Create enum which will display unified output status (`ERROR`, `NORMAL` or smth else), then based on value you'll print text with specified color.